features/ios/support/helper.rb

```diff
@@ -0,0 +1,202 @@
+=begin
+
+These methods are implemented for Calabash-android.
+We are implementing their iOS equivalents if possible so some of them may be empty.
+
+=end
+
+
```

Comment:
Remove spaces.

Reply:
Actually, remove this entire file.  These methods are implemented in Calabash iOS?